samples/excel/89-preview-apis/chart-title-substring.yaml

```diff
@@ -0,0 +1,126 @@
+id: excel-chart-title-substring
+name: Chart - Title substring
+description: Get and set title substring for a chart.
+host: EXCEL
+api_set:
+    ExcelApi: 1.7
+script:
+    content: |+
+        $("#setup").click(() => tryCatch(setup));
+        $("#change-title-substring").click(() => tryCatch(changeTitleSubstring));
+
+        async function changeTitleSubstring() {
+            await Excel.run(async (context) => {
+                let sheet = context.workbook.worksheets.getItem("Sample");
+                let chart = sheet.charts.getItemAt(0);
+             
+                // Get first seven characters of the title and highlight them to green.
+                let font = chart.title.getSubstring(0, 7).font;
+                font.color = "Green";       
+            });
```

Comment:
Is there a reason for not ending the Excel.run with a context.sync the way we usually do?